FreeRTOS/FreeRTOS-Kernel
# Add missing xPortIsInsideInterrupt function

Description
-----------
The xPortIsInsideInterrupt function is missing from the ARM Cortex-M0 ports. Add this function.

Test Steps
-----------


Checklist:
----------


- [ X ] I have tested my changes. No regression in existing tests.
- [ N/A ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
